src/pages/Swap.tsx

```diff
@@ -514,7 +557,8 @@ function Swap(): ReactElement {
   }
 
   async function handleConfirmTransaction(): Promise<void> {
-    const fromToken = tokenSymbolToTokenMap[formState.from.symbol]
+    if (!tokens) return
```

Comment:
We should display an error to the user instead of failing silently when they take an action.